research-software-directory/RSD-as-a-service
# feat: pinned projects use icons instead of color

# Pinnen projects and software use icon instead of background color

Closes #461 

Changes proposed in this pull request:

* Apply icon before the title for featured software and project at organisation 

Note! Ewan and me choose specific icon but any icon from [material ui icons set](https://mui.com/material-ui/material-icons/) could be applied

How to test:

* `docker-compose build frontend` to build
* `docker-compose` up to start
* login as rsd_admin and visit an organisation with software and projects
* pin the software or project and confirm that icon is shown. The items will also be first in the list

## Example pinned project
First 2 projects are pinned and the last one is not yet published.

![image](https://user-images.githubusercontent.com/9204081/195039782-7bc435bd-8f4c-42c2-9598-43cff0f98cfd.png)

PR Checklist:
